### DISCUSSION TITLE
github avatars

### DISCUSSION THREAD

**Discussion by conifercone** [community/community#133465] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/5504c580-fdee-4c88-b78c-31668b214318)
![image](https://github.com/user-attachments/assets/403b00d4-1671-4eaa-aeae-8920a524cc3b)
hi,Why is the avatar in the settings not updated correctly?

**Comment by LuaanNguyen** [comment, marked as answer]
Hi @conifercone, 

Give it a minute, or try to refresh the page. This can sometimes take a little bit to upload. You profile is correct on my end though so it shouldn't be a problem.

**Reply by conifercone** [reply]
Thanks for the reply, I'll wait and see

**Comment by Bernardo-Zamin** [comment]
Here for me your icon appears correctly

**Comment by conifercone** [comment]
I found that the request parameters cannot be obtained if the size is added
```shell
https://avatars.githubusercontent.com/u/30498960?v=4
https://avatars.githubusercontent.com/u/30498960?s=96&v=4
```

**Reply by conifercone** [reply]
interesting🤣

**Reply by Bernardo-Zamin** [reply]
Now it works ?

**Reply by conifercone** [reply]
no😅

**Reply by conifercone** [reply]
I don't know why I can't get the correct image when request the same ID and add the size.

**Reply by conifercone** [reply]
I'm waiting to see

**Reply by Bernardo-Zamin** [reply]
I have no idea how to solve it, have you tried to ask ChatGPT? lol🤣

**Comment by 1uv0cean** [comment]
It requires time to apply changes.
